### PR TITLE
cloud_firestore

### PR DESCRIPTION
**Pull request by harshsinghs1058**
added necessary changes for cloud store

### CODE REVIEW

**Comment by arteevraina** [comment]
@harshsinghs1058 You need to read contribution guidelines.

File: android/app/build.gradle [hunk 1]
```diff
@@ -40,7 +40,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.knowyourdonor"
-        minSdkVersion 16
```
Comment (arteevraina) [inline android/app/build.gradle:43]: Did you try running the app after making this change ?

Comment (harshsinghs1058) [reply android/app/build.gradle:43]: their are some spelling mistakes in the main flutter code..should i also correct them???

Comment (arteevraina) [reply android/app/build.gradle:43]: You can make separate issue for this and send a separate PR for that also.

Comment (harshsinghs1058) [reply android/app/build.gradle:43]: this app is  running fine

Comment (arteevraina) [reply android/app/build.gradle:43]: Cool! I have just added the static UI for the app.

Comment (harshsinghs1058) [reply android/app/build.gradle:43]: Why my pull request is not still not accepted?

Comment (harshsinghs1058) [reply android/app/build.gradle:43]: Why my pull request is not still accepted?

Comment (arteevraina) [reply android/app/build.gradle:43]: You need to write meaningful commit messages. You can have a look at CONTRIBUTING.MD for that.

**Pull request closed without merging**